From 02cd089fa4ac175a64624c8820068700fbda2db8 Mon Sep 17 00:00:00 2001
From: Manuel Bernhardt <bernhardt.manuel@gmail.com>
Date: Wed, 17 Aug 2011 16:08:01 +0200
Subject: [PATCH] Not generating empty constructor for scala

---
 .../classloading/enhancers/PropertiesEnhancer.java |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/framework/src/play/classloading/enhancers/PropertiesEnhancer.java b/framework/src/play/classloading/enhancers/PropertiesEnhancer.java
index 09ae599..3ceb3dd 100644
--- a/framework/src/play/classloading/enhancers/PropertiesEnhancer.java
+++ b/framework/src/play/classloading/enhancers/PropertiesEnhancer.java
@@ -39,6 +39,9 @@ public class PropertiesEnhancer extends Enhancer {
         }
 
         // Add a default constructor if needed
+        if(isScala(applicationClass)) {
+            return;
+        }
         try {
             boolean hasDefaultConstructor = false;
             for (CtConstructor constructor : ctClass.getDeclaredConstructors()) {
-- 
1.7.5.4

